fedora_messaging/cli.py

```diff
@@ -383,7 +383,7 @@ def publish(exchange, file):
             click.echo("Publishing message with topic {}".format(msg.topic))
             try:
                 api.publish(msg, exchange)
-            except exceptions.PublishReturned as e:
+            except (exceptions.PublishReturned, exceptions.PublishForbidden) as e:
```

Comment:
I'd except the generic PublishException here since all it's going to do is report the error and doesn't differentiate on the type.